### PR TITLE
java snips

### PR DESCRIPTION
**Pull request by wsdjeg**
goals:
- [ ] translate [vim-snippets](https://github.com/honza/vim-snippets) to this repo

### CODE REVIEW

File: neosnippets/java.snip [hunk 1]
```diff
@@ -1,18 +1,45 @@
...
 snippet interface
     interface ${1} ${2:extends} ${3:#:Parent} {
         ${0:TARGET}
     }
 
-snippet method
-    ${1:void} ${2:#:method}(${3}) ${4:throws} {
-        ${0:TARGET}
+snippet     method
+abbr        m
```
Comment (Shougo) [inline neosnippets/java.snip]: Why the abbr is m?

Comment (wsdjeg) [inline neosnippets/java.snip]: sorry i just want to add alias,i will delete it ,this PR has not been finished

**Comment by Shougo** [comment]
What is the status?

**Comment by wsdjeg** [comment]
sorry, it is stopped for a year. I will start to work on it.

**Comment by Shougo** [comment]
OK

**Comment by Shougo** [comment]
Any progress?

**Comment by wsdjeg** [comment]
sorry，I can not work on this now.

**Comment by Shougo** [comment]
OK

**Pull request closed without merging**